app/models/middleware_server.rb

```diff
@@ -1,5 +1,6 @@
 class MiddlewareServer < ApplicationRecord
   belongs_to :ext_management_system, :foreign_key => "ems_id"
+  belongs_to :middleware_server_group, :foreign_key => "server_group_id"
```

Comment:
please check if this works for standalone servers which don't belong to a server group.
I think that it won't work without specifying here `optional: true` as Rails 5 expects (and validates) by default the relationship to be present.